docs/revanced-cli.md

```diff
@@ -1,40 +1,39 @@
+Requirements
+==
+
+- An Android 8+ device (to install the patched APK on)
```

Comment:
For CLI commands, there are no restrictions on the Android version of the device to use the command.

Apps supported by the RVX patch are `YouTube`, `YouTube Music`, and `Reddit`, which require Android 8.0+

Reply:
> But.. apps supported by the RVX patch are `YouTube`, `YouTube Music`, and `Reddit`, which require Android 8.0+

That's what I was thinking. Should I make this reasoning more clear in the doc?

Reply:
The meaning isn't much different, so I don't think there's a need to add an explanation